sdk/core/core-http/samples/index.js

```diff
@@ -0,0 +1,26 @@
+const { ServiceClient } = require("@azure/core-http");
+
+const subscriptionId = "<subscription id>";
+const token = "<access token>";
+
+class TestTokenCredential {
+  constructor(token, expiresOn) {
+    this.token = token;
+    this.expiresOn = expiresOn ? expiresOn.getTime() : Date.now() + 60*60*1000;
+  }
+  async getToken(_scopes,_options) {
+    return {
+      token : this.token,
+      expiresOnTimestamp : this.expiresOn
+    }
+  }
+}
+
+document.write('hello world');
```

Comment:
document.write is considered an anti-pattern

Reply:
replaced with console.log